blockchain/tx_pool.go

```diff
@@ -637,9 +643,31 @@ func (pool *TxPool) validateTx(tx *types.Transaction) error {
 	}
 
 	// NOTE-Klaytn Drop transactions with unexpected gasPrice
-	if pool.gasPrice.Cmp(tx.GasPrice()) != 0 {
-		logger.Trace("fail to validate unitprice", "Klaytn unitprice", pool.gasPrice, "tx unitprice", tx.GasPrice())
-		return ErrInvalidUnitPrice
+	// If the transaction type is DynamicFee tx, Compare transaction's GasFeeCap(MaxFeePerGas) and GasTipCap with tx pool's gasPrice to check to have same value.
+	if tx.Type() == types.TxTypeDynamicFee {
+		// Sanity check for extremely large numbers
+		if tx.GasTipCap().BitLen() > 256 {
+			return ErrTipVeryHigh
+		}
+
+		if tx.GasFeeCap().BitLen() > 256 {
+			return ErrFeeCapVeryHigh
+		}
```

Comment:
We don't need below logic? 
I think we don't need with fixed gas price for now, but this will be used when we do not use fixed gas price.
And this logic looks reasonable with now our policy also, so how about adding this? (Not same, but check logic for making sure gasFeeCap is bigger or equal than gasTipCap)
```
	// Ensure gasFeeCap is greater than or equal to gasTipCap.
	if tx.GasFeeCapIntCmp(tx.GasTipCap()) < 0 {
		return ErrTipAboveFeeCap
	}
```